.github/workflows/ci.yml

```diff
@@ -31,7 +31,8 @@ jobs:
                 sudo apt install --no-install-recommends \
                   libdbus-1-dev libgirepository1.0-dev gir1.2-gtk-3.0 gir1.2-notify-0.7 gir1.2-gudev-1.0 graphviz \
                   imagemagick libpulse-dev lm-sensors git xserver-xephyr xterm xvfb x11-apps ninja-build libegl1-mesa-dev \
-                  libgles2-mesa-dev libgbm-dev libinput-dev libxkbcommon-dev libpixman-1-dev libpciaccess-dev
+                  libgles2-mesa-dev libgbm-dev libinput-dev libxkbcommon-dev libpixman-1-dev libpciaccess-dev \
+                  dbus-x11 libnotify-bin
```

Comment:
In what way is `dbus-x11` required, compared to other packages, presumably `dbus`? I haven't changed anything in my installation and notifications through dbus work normally in Wayland.

Reply:
It provides `dbus-launch` which the test uses to create a separate dbus session.

Reply:
RIght yeah, strange that it isn't provided by dbus directly considering it's also applicable to wayland environments. Fair enough.